packages/js/product-editor/src/blocks/section/block.json

```diff
@@ -14,6 +14,11 @@
 		"description": {
 			"type": "string",
 			"__experimentalRole": "content"
+		},
+		"blockGap": {
+			"type": "string",
+			"enum": [ "lg", "2xlg" ],
```

Comment:
We discussed this in our grooming call, but this is the only area the feels a little weird with the naming.

It would be great if we could match the gaps to our existing variable gap sizes (24px, 36px, 40px) and then match the naming there as well.  @jarekmorawski is that feasible or do these gaps need to be different than our current gaps?

Reply:
Meaning that we'd change all the gaps between fields and sections to those you listed? I'd prefer to avoid that. The ones in Figma are consistent with WordPress (more or less) and balance white space and density well. For example, increasing the gap between sections from 32 px to 36 px or 40 px will impact how much content is visible before scrolling.

By the way, where do our current gap sizes come from and can we adjust them to the new ones?

Reply:
Those gaps came from the previous designers, but I'm unsure on the specifics of how they were chosen.

It would be really great to adjust them, though that would require changes in many places (a quick scan shows that we're using some form of the `$gap` variable in 762 files).

But if this is something we plan on doing one day, we could at least update the naming in this PR to match those.  @jarekmorawski do you have ideal sizes for gaps/spacing for smallest, smaller, small, default, large, larger, and largest?

Reply:
If we update the gaps, it'll affect all of wc-admin, correct? We'd have to do an audit to make sure we wouldn't break the visual rhythm outside the form, so I can't specify them yet. For the form, let's use the sizes below.

I'll leave it to your judgment and BEM best practices regarding naming.

- XXS: 8px
- XS: 16px
- S: 24px
- M: 32px
- L: 40px
- XL: 48px
- XXL: 56px

We'll unlikely ever use gaps larger than 56 px for form elements or cards like in the Home screen. The only things I can think of are margins and paddings, but I'm not sure we should include them in the system.

Reply:
>If we update the gaps, it'll affect all of wc-admin, correct?

Correct, but we're not looking at tackling that in this PR.  Mostly just curious if it's something we'll eventually make sizes consistent on.  If so, we can match the naming and eventually replace them.

On the other hand, providing new sizes (xxs, xs, s, etc) provides us an easier way to transition over.  I'm indifferent between those sizes and using the previous semantics (smallest, smaller, small, etc).  @mdperez86 do you have an opinion on this?

Reply:
done -> https://github.com/woocommerce/woocommerce/pull/38325/commits/ab0329f9cbd4ccf357a5338fdcac14e1fd8cccf6